scripts/build_ammonia_production.py

```diff
@@ -42,15 +43,15 @@
         skiprows=5,
         header=0,
         index_col=0,
-        skipfooter=19,
+        skipfooter=7,
         na_values=["--"],
     )
 
     ammonia.index = cc.convert(ammonia.index, to="iso2")
 
-    years = [str(i) for i in range(2013, 2018)]
+    years = [str(i) for i in range(2018, 2023)]
 
-    ammonia = ammonia[years]
+    ammonia = ammonia.rename(columns=lambda x: str(x))[years]
```

Comment:
Could then just be shorter, right?

```py
ammonia = ammonia[list(range(2018, 2023))]
```

Reply:
mhm, the columns of the ammonia dataframe are mixed type, so some of the years are read in as `int` while others a read in as `str`. Therefore, I would keep it, as it is for now
```
ammonia.columns
Index([ 'Unnamed: 1',          2018,  'Unnamed: 3',          2019,
        'Unnamed: 5',        '2020',  'Unnamed: 7',        '2021',
        'Unnamed: 9',        '2022', 'Unnamed: 11'],
      dtype='object')
```